commodore/config.py

```diff
@@ -34,6 +34,12 @@ def __init__(self, api_url, api_token, global_git, verbose):
         self._components = {}
         self._config_repos = {}
         self._verbose = verbose
+        
+        self.username = None
```

Comment:
These assignements to `None` and the following `if is not None` seems unnecessary, since the default for these parameters is `None` anyway.